php/php-mailparse/Portfile

```diff
@@ -34,6 +33,7 @@ long_description \
     It can deal with rfc822 and rfc2045 (MIME) compliant messages.
 
 if {${name} ne ${subport}} {
+    depends_build       port:autoconf port:${php}-mbstring
```

Comment:
I needed to add `mbstring` as a build dependency as trace mode showed that it was trying to access `/opt/local/include/php73/php/ext/mbstring/libmbfl/mbfl/mbfilter.h`. Adding simply `depends_build port:${php}-mbstring` then errored on a requirement for `autoconf`, so that's added too.